django/forms/widgets.py

```diff
@@ -181,6 +181,7 @@ class Widget(six.with_metaclass(MediaDefiningClass)):
     needs_multipart_form = False  # Determines does this widget need multipart form
     is_localized = False
     is_required = False
+    supports_microseconds = True
```

Comment:
This PR should now only be about documentation with [this commit](https://github.com/django/django/commit/4fc27871fe433cbe7344b8898cb05abd0785d239).

There is now a new PR #4467 for the code and tests. I added a TextInput case in that PR.